src/src/PluginStructs/P092_data_struct.h

```diff
@@ -1,5 +1,5 @@
-#ifndef DLBus_H
-#define DLBus_H
+#ifndef PLUGINSTRUCTS_P036_DATA_STRUCT_H
+#define PLUGINSTRUCTS_P036_DATA_STRUCT_H
```

Comment:
Please use P092 instead of P036, or else you will get very strange compile results when using both plugins :)